src/client.coffee

```diff
@@ -269,7 +282,7 @@ module.exports = class Client
     Report transaction(s).
 
     Parameters:
-      service_id {String} Optional (required only if you have more than one service)
+      service_id {String} Required (from November 2016)
```

Comment:
You forgot the service id in the example below: `client.report trans, (response) ->`